front/app/api/analysis_stats/useAuthorsByAge.ts

```diff
@@ -0,0 +1,37 @@
+import { useQuery } from '@tanstack/react-query';
+import { CLErrors } from 'typings';
+import fetcher from 'utils/cl-react-query/fetcher';
+import {
+  IAuthorsByAge,
+  AuthorsByAgeKeys,
+  AuthorsByAgeQueryParams,
+} from './types';
+import { authorsByAgeKeys } from './keys';
+
+const fetchUsersByAge = (
+  analysisId: string,
+  queryParams: AuthorsByAgeQueryParams
+) =>
+  fetcher<IAuthorsByAge>({
+    path: `/analyses/${analysisId}/stats/authors_by_age`,
+    action: 'get',
+    queryParams,
+  });
+
+const useAuthorsByAge = ({
+  analysisId,
+  enabled,
+  queryParams,
+}: {
+  analysisId: string;
+  enabled?: boolean;
+  queryParams: AuthorsByAgeQueryParams;
+}) => {
+  return useQuery<IAuthorsByAge, CLErrors, IAuthorsByAge, AuthorsByAgeKeys>({
+    queryKey: authorsByAgeKeys.item({ analysisId, params: queryParams }),
+    queryFn: () => fetchUsersByAge(analysisId, queryParams),
+    enabled: typeof enabled == undefined ? true : enabled,
```

Comment:
This will not work - `undefined` needs to be a string. But also, perhaps cleaner is to set a default value for the hook prop above `enabled=true`. Not 100% sure but I think we have some examples on how to do this in other hooks.

Reply:
Good catch :) The argument is actually unused, took this along with copy/paste and modified it wrongly, so I'll just take it out altogether.